### PR TITLE
Allow override of invite template

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* Add UI for admins to override  the realm invitation template

invitetemplate

**Release Note**

```release-note
Allow realm custom template for invitations
```

### CODE REVIEW

**Comment by whaught** [comment]
I'm considering setting the element to disabled if SMTP creds aren't set?

File: cmd/server/assets/realmadmin/_form_email.html [hunk 1]
```diff
@@ -79,6 +79,41 @@
         587 is the default port for SMTP, and legacy port 25 is blocked.
       </small>
     </div>
+
+    <div class="form-label-group">
+      <textarea name="email_invite_template" id="email-invite-template" class="form-control text-monospace{{if $realm.ErrorsFor "SMSTextTemplate"}} is-invalid{{end}}"
```
Comment (sethvargo) [inline cmd/server/assets/realmadmin/_form_email.html:84]: Still refs SMS here

**Review by sethvargo — CHANGES_REQUESTED** [review]

File: cmd/server/assets/realmadmin/_form_email.html (hunk 1 above)
Comment (whaught) [reply cmd/server/assets/realmadmin/_form_email.html]: thx

**Review by sethvargo — APPROVED** [review]
/lgtm

**Pull request merged**